src/app/backend/systembanner/api/types.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+const (
+	// SystemBannerSeverityInfo is the lowest of allowed system banner severities.
+	SystemBannerSeverityInfo = "INFO"
+
+	// SystemBannerSeverityInfo is in the middle of allowed system banner severities.
+	SystemBannerSeverityWarning = "WARNING"
+
+	// SystemBannerSeverityInfo is the highest of allowed system banner severities.
+	SystemBannerSeverityError = "ERROR"
+)
+
+// SettingsManager is used for user settings management.
+type SettingsManager interface {
+	// Get system banner.
+	Get() *SystemBanner
+}
+
+// SystemBanner represents system banner.
+type SystemBanner struct {
+	Message  string `json:"message"`
+	Severity string `json:"severity"`
+}
+
+// GetSeverity returns one of allowed severity values based on given parameter.
+func GetSeverity(severity string) string {
```

Comment:
I'd add type that maps to a string (i.e. `type BannerSeverity string`) so we can return our type here. It's easier to find out correct values later on.

Maybe also use a switch on string here as it feels like a perfect place for it.

Reply:
+1

Reply:
Done.